__tests__/Ship.test.js

```diff
@@ -0,0 +1,64 @@
+/* globals describe it expect */
+const Ship = require('../src/Ship.js')
+const Port = require('../src/Port.js')
+const Itinerary = require('../src/Itinerary.js')
+
+describe("Ship", () => {
+   describe("with ports and an itinerary", () => { 
+    let ship;
+    let dover;
+    let calais;
+    let itinerary;
+  
+    beforeEach(() => {
+      dover = // new Port('Dover');
+      { addShip :jest.fn() ,removeShip:jest.fn() ,name : 'Dover', ships : [] };
+
+      calais = // new Port('Calais');
+      calais = {addShip :jest.fn() ,removeShip:jest.fn() ,name : 'Calais', ships : [] }; 
+
+      itinerary = new Itinerary([dover, calais]);
+      ship = new Ship(itinerary);
+    });
```

Comment:
This should be re-written:
```
    beforeEach(() => {
      dover = {
        addShip: jest.fn(),
        removeShip: jest.fn(),
        name: "Dover",
        ships: [],
      };

      calais = {
        addShip: jest.fn(),
        removeShip: jest.fn(),
        name: "Calais",
        ships: [],
      };

      itinerary = new Itinerary([dover, calais]);
      ship = new Ship(itinerary);
    });
```
Aside fro the formatting being a bit messy, the comments should be taken out